lib/msw-config.ts

```diff
@@ -1,9 +1,63 @@
-import { rest, setupWorker } from 'msw';
+import {
+  rest,
+  RestHandler,
+  setupWorker,
+  type RestRequest,
+  type SetupWorkerApi,
+} from 'msw';
+import type { Server } from 'miragejs';
```

Comment:
This type isn't actually used, but because of the way the mirage types are defined, the `/server` types won't be found unless _something_ is imported from the top level, in which case the other modules are cached and used as well.